app/mailers/user_mailer.rb

```diff
@@ -16,6 +16,14 @@ def notification(title: '', body: '', path: '', emails: [])
          to: emails
   end
 
+  def form_feedback(form_id:, email:)
+    set_logo
+    @form = Form.find(form_id)
```

Comment:
When a user submits the "About your experience using Touchpoints" form, will there be any way to tell which archived form they are submitting it about? Would it be useful to know?

Reply:
good call. will add ?location_code